tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -955,6 +969,14 @@ void test() {
 
     test_zoned_time_formatter<char>();
     test_zoned_time_formatter<wchar_t>();
+
+#if !defined(_DLL) || _ITERATOR_DEBUG_LEVEL == DEFAULT_IDL_SETTING
+    test_locale<wchar_t>();
+#ifndef MSVC_INTERNAL_TESTING // TRANSITION, the Windows version on Contest VMs doesn't always understand ".UTF-8"
+    setlocale(LC_ALL, ".UTF-8");

```

Comment:
I think we should also verify that `setlocale()` succeeds:

https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/tests/std/tests/P0645R10_text_formatting_utf8/test.cpp#L157